src/scss/components/_mainImage.scss

```diff
@@ -0,0 +1,18 @@
+.mainImageContainer{
+    width: 250px;
+    height: 350px;
+    background-color: silver;
+    border: 2px #404040;
+    position: relative;
+    top: 50%;
+    left: 25%;
```

Comment:
Przesunąłbym może bliżej lewej krawędzi, ale według własnego uznania, napisz co sądzisz i daje approve

Reply:
To jeszcze będzie przesuwane w przyszłych taskach. Na razie tylko, żeby miało to ręce i nogi.